test/integ/quote-gouda.test.ts

```diff
@@ -96,6 +102,28 @@ const checkQuoteToken = (
   expect(percentDiff.lessThan(new Fraction(parseInt(SLIPPAGE), 100))).to.be.true;
 };
 
+const checkPortionRecipientToken = (
```

Comment:
just confirming that this will be part of main once we push the classic code right?